bench/wtperf/wtperf_run_py/perf_stat_collection.py

```diff
@@ -49,21 +50,29 @@ def __init__(self):
     def add_stat(self, perf_stat: PerfStat):
         self.perf_stats[perf_stat.short_label] = perf_stat
 
-    def find_stats(self, test_stat_path: str, operations: List[str]):
+    def find_stats(self, test_stat_path: str, operations: List[str], run_max: int):
```

Comment:
Is `run_max` here necessary?

Reply:
Nope, good catch!
This is left over from an assertion I was using but decided to remove. run_max is removed in 5e1f2c4